website/docs/docs/user-interface/settings/users.md

```diff
@@ -2,84 +2,83 @@
+_NOTE_: If the user being invited (as identified by their email address) already exists in PactFlow, they **will not** receive another email and can login to your account with their existing credentials.
 
 ## User roles
 
 By default, each user is assigned the `User` role. The account creator is also assigned the `Administrator` role. See [Roles and Permissons](/docs/permissions/predefined-roles) for more information.
 
-There are a few options to change a user's roles. The `Make Administrator`/`Remove Administrator` menu items allow the administrator role to be added or removed. `Edit Roles` allows all the roles of a user to be edited. The `Add Role`/`Remove Role` bulk action menu items can add or remove a role from multiple users.
+There are a few options to change a user's role. The `Make Administrator`/`Remove Administrator` menu items allow administrators role to be added or removed. `Edit Roles` allows all the roles of a user to be edited. The `Add Role`/`Remove Role` bulk action menu items can add or remove a role from multiple users.
 
 ## System accounts
 
-You can have a number of system accounts, based on your plan. System accounts are a type of user account to be used with API
-access only. They are primarily for use with CI/CD systems to access PactFlow. From the system account screen (by selecting it from the user type dropdown), you can create new system accounts, disable existing ones and get or renegerate the access tokens for the account. These functions require administrator access. You can also see the last time a token for that account was
+You can have multiple system accounts, based on your plan. System accounts are a type of user account to be used only with API access. They are primarily for use with CI/CD systems to access PactFlow. From the system account screen (by selecting it from the user type dropdown), you can create new system accounts, disable existing ones and get or renegenerate the access tokens for the account. These functions require administrator access. You can also see the last time a token for that account was
 used to access any PactFlow API.
 
 ![System accounts](/ui/system-accounts.png)
 
 ## Federation with Google, GitHub or SAML2.0
 
-Federation allows team members to join your account without being invited - i.e. they will be provisioned into PactFlow on demand, up until your account limit. You can simply share the link to the account `https://<account>.pactflow.io` and they may choose to login with Google, GitHub or your SAML provider.
+The Federation allows team members to join your account without being invited - i.e., they will be provisioned into PactFlow on demand, up until your account limit. You can simply share the link to the account `https://<account>.pactflow.io` and they may choose to login with Google, GitHub or your SAML provider.
```

Comment:
"Federation" is an adjective here, so it need not be capitalised. "The" Federation therefore doesn't make sense either. Suggest:

```suggestion
Federation allows team members to join your account without being invited - i.e., they will be provisioned into PactFlow on demand, up until your account limit. You can simply share the link to the account `https://<account>.pactflow.io` and they may choose to login with Google, GitHub or your SAML provider.
```